docs/contributing/create-edit-doc.md

```diff
@@ -0,0 +1,179 @@
+
+:::sumo Micro Lesson
+Check out this brief tutorial on how to submit a basic change to our docs.
+
+<Iframe url="https://www.youtube.com/embed/2ONgNj8pNLY?rel=0"
+        width="854px"
+        height="480px"
+        id="myId"
+        className="video-container"
+        display="initial"
+        position="relative"
+        allow="accelerometer; autoplay=1; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+        />
+
+        <br/>
+
+<details>
+<summary>View text instructions</summary>
+
+1. Scroll to the bottom of that doc and click the **Edit this page** link. This will open your selected doc in **Edit file** mode on our GitHub repo website.
+1. Click **Fork this repository** to continue.
+1. Apply your edits to the file.
+1. Scroll down a bit on the page until you see the description field, enter a brief summary of your changes there, then click **Commit changes**.
+1. In the **Propose changes** dialog, enter a description of your change, enter a new name for your branch if desired, and click **Propose Change**.
+
+This will fork and submit changes to the Docs Team for review.
+</details>
+
+:::
+
+
+## Edit a doc
+
+### Step 1: Fork the Sumo Docs repository
+
+1. Fork the [Sumo Docs repository](https://github.com/SumoLogic/sumologic-documentation) locally. Remember to sync your fork and update branches as needed.
+   :::tip GitHub tips
+   * [How to fork a repo](https://help.github.com/articles/fork-a-repo/)
+   * [How to sync your fork with branches](https://help.github.com/articles/syncing-a-fork/)
+   :::
+1. Review our [README](https://github.com/SumoLogic/sumologic-documentation#readme) documentation guidelines.
+1. Create a new branch from your forked repo using a name that best describes the work or references a GitHub issue number. For example, if you wanted to submit a PR to edit our Elasticsearch app doc, you'd write something like: `<your initials>-apps-elasticsearch`.
+
+import Tools from '../reuse/contributing/tools.md';
+
+<Tools/>
+
+### Step 2: Edit your doc
+
+In your new branch, edit the doc markdown file. See our [Style Guide](/docs/contributing/style-guide) to learn how to style content, add code snippets, import multimedia, and more. Doc body text content is written in GitHub-flavored markdown, with some customizations.
+
+### Step 3: Preview your changes
+
+<Preview/>
+
+### Step 4: Submit your request
+
+<Submit/>
+
+## Create a new doc
+
+To submit more extensive edits, such as creating a new doc, we recommend forking our repo, making changes in a new branch, and submitting a PR for review.
+
+Feel free to reach out to us, the Docs Team, to discuss. We're happy to work with you on the project and talk through rewriting content, changing flow, adding a new topic or section, and deprecating content.
```

Comment:
How should they reach out to us? Should we include an email address?

Reply:
updated!